uni/ulsp/database.icn

```diff
@@ -112,15 +123,23 @@ class LSPDB(
 
       Logger.debug1_kind("db", "Looking in unicon paths")
       every dir := !db_paths do {
+         dir := map(dir, "\\", "/")
+         put(dir_list, dir)
          Logger.debug2_kind("db:path", "Processing ", dir)
          build_by_path(dir)
       }
+      put(dir_list, unicon_dir || "uni/ulsp")
    end
 
    method build_by_path(path)
       local idoc, x, pack, file_in_pack, file_without_ext, import_in_file, procedure_in_file, param_in_procedure,
          paramName, paramType, paramDef, class_in_file, method_in_class, param_in_method, class_param, class_comment,
-         constructor_param
+         constructor_param, package_table, file_table, file_wo_ext_table, import_table, procedure_table, procedure_in_file_table,
+         param_table, classes_table, class_in_file_table, constructor_table, method_table, attributes_table, comments_table,
+         superclass_table, src_table, cons_name_table, cons_comments_table, cons_params_table, method_name_table, method_name_comments_table,
+         method_name_params_table
+
```

Comment:
Can you make the lines shorter by moving some of the variables to the bottom short line?